src/foam/u2/crunch/wizardflow/RequestNotificationPermissionAgent.js

```diff
@@ -59,6 +63,10 @@ foam.CLASS({
         {
           class: 'String',
           name: 'prefix'
+        },
+        {
+          class: 'Boolean',
+          name: 'hideDontShowAgain'
```

Comment:
I dont think we need this change in this file, they can select dont show again here

Reply:
If they select it, there is an entry in localStorage called 'refusedNotification' that you can check in your predicate and return false if it has been set to true